README.rdoc

```diff
@@ -105,6 +105,12 @@ Example configuration:
       multiline_flush_interval 0.1
     </match>
 
+== Notes
+
+New language support requires adding new rules in this gem. But that alone only
+help bundle multi-line messages into one line. Logging backend will need to
+support this new exception as well for proper recognization down the pipeline.
```

Comment:
I think a more useful message would tell the reader what to do. It's not clear to me how one gets the Logging backend to support this, or if that's even something a curious reader will be able to do.

Reply:
Added some instruction on how to file feautre request support case. Also added an example (Dart language support) for clarification.